spec/support/mock_rails.rb

```diff
@@ -121,18 +121,18 @@ def new_author_path(*_args)
     allow(@author).to receive(:favorite_color).and_return('#424242')
     allow(@author).to receive(:favorite_book).and_return(nil)
 
-    @book_0 = ::Book.new
-    allow(@book_0).to receive(:id).and_return('78')
-    allow(@book_0).to receive(:title).and_return("Gulliver's Travels")
-    @book_1 = ::Book.new
-    allow(@book_1).to receive(:id).and_return('133')
-    allow(@book_1).to receive(:title).and_return('Treasure Island')
-    @genre_0 = ::Genre.new
-    allow(@genre_0).to receive(:name).and_return('Exploration')
-    allow(@genre_0).to receive(:books).and_return([@book_0])
-    @genre_1 = ::Genre.new
-    allow(@genre_1).to receive(:name).and_return('Pirate Exploits')
-    allow(@genre_1).to receive(:books).and_return([@book_1])
+    @book0 = ::Book.new
```

Comment:
Thanks for fixing the linting errors, however we have not discussed how we want to display numbered variables.

@dgmstuart If you are not aware, a [Rubocop rule](https://github.com/bbatsov/rubocop/pull/3416) was added to enforce how numbers are displayed in variables.  There are three options:

1. normal: `book1`
2. snake_case: `book_1`
3. non_integer: `book_one`

I prefer `snake_case` and it seems like that is what we are currently using.  To change from the default `normal`, we can add
```
Style/VariableNumber:
  EnforcedStyle: 'snake_case'
```
to our Rubocop config.  What is your opinion?